lib/chef/provisioning/aws_driver/aws_resource.rb

```diff
@@ -60,6 +60,23 @@ def aws_object
     raise NotImplementedError, :aws_object
   end
 
+
+  #
+  # Is this object ready to be tagged?
+  #
+  def taggable?
+    false
+  end
+
+  #
+  # Wait until this object.taggable? returns true
+  #
+  def wait_until_taggable()
+    Retryable.retryable(:tries => 12, :sleep => 5, :on => [AWS::EC2::Errors::InvalidInstanceID::NotFound, TimeoutError]) do
+      raise TimeoutError unless taggable?
```

Comment:
Doesn't this call the `taggable?` on line 67, which is always `false`? I'm not seeing the mechanism by which [this method call](https://github.com/chef/chef-provisioning-aws/pull/219/files#diff-77d92b3d739e913b64448070c03fa0c0R54) (`new_resource.wait_until_taggable`) calls [this `taggable?`](https://github.com/chef/chef-provisioning-aws/pull/219/files#diff-77d92b3d739e913b64448070c03fa0c0R42) in `Chef::Provider::AwsVpc` instead of the always-`false` one.